paper.tex

```diff
@@ -1422,6 +1424,14 @@ \section*{ARG inference methods}\label{ARG_inference_methods}
 evaluation of sampling probabilities. While not competitive with \texttt{ARGWeaver}, their method extends
 the feasible range of sequence lengths by at least an order of magnitude.
 
+\subsection*{Differences in reconstructed ARGs}
+
+There are a vast number of ARGs compatible with a given dataset, so ARG reconstruction tools generally produce different outputs for the same input. Figure \ref{fig-inferred-args} shows examples of the ARG topologies produced by four of the tools mentioned above, applied to the classical dataset of 11 \emph{Drosophila melanogaster} sequences of \citet{kreitman1983nucleotide}.
+
+\texttt{KwARG} \citep{ignatieva2021kwarg} produces a parsimonious ARG with seven recombination breakpoints, which has been shown to be the true minimum required to explain the dataset when disallowing recurrent mutation \citep{song2003parsimonious}. An ARG drawn from the posterior sample produced by \texttt{ARGweaver} has 17 breakpoints. Both tools explicitly record the sequence undergoing each recombination event, and the precise location of the breakpoint along the genome. \texttt{Relate} \citep{speidel2019method} and \texttt{tsinfer} \citep{kelleher2019inferring} instead construct a sequence of correlated local trees, but without specifying the precise positions of recombination events (i.e.\ reconstructing the local trees without explicitly specifying the SPR operations required to transition between them). The number of inferred trees also depends on the parameter settings; all four tools can allow for the presence of sequencing errors and recurrent mutations, reducing the number of inferred local trees (in this case, based on the input mutation and recombination rate \texttt{Relate} hence posits only two recombination breakpoints).
```

Comment:
I worry that saying 

> "Relate" and "tsinfer" instead construct a sequence of correlated local trees

will reinforce the (mistaken) assumption that these are both JBOTs. I would quite like to rephrase this?

Reply:
How about

> \texttt{Relate} \citep{speidel2019method} and \texttt{tsinfer} \citep{kelleher2019inferring} do not place recombination events onto the ARG topology, so do not encode the transitions between local trees in terms of explicit SPR operations.

Reply:
Cheers, I think this is more to the point! 
Regarding Relate, I've said
> \texttt{Relate} first infers local tree topologies and then identifies edges that persist from one tree to the next as a separate step, while \texttt{tsinfer} explicitly constructs nodes that are shared across local trees.
I think in general Relate is pretty good at identifying equivalent branches.

Reply:
Yes, that sounds good to me. I do think in the figure that the Relate ARG looks rather different from the others, rather like a series of trees, so maybe worth commenting on somewhere else?